SB.js

```diff
@@ -0,0 +1,67 @@
+SB = {};
+
+function configProxy() {
+    chrome.storage.onChanged.addListener((changes, namespace) => {
+        for (key in changes) {
+            SB.localconfig[key] = changes[key].newValue;
+        }
+    });
+    var handler = {
+        set: function(obj, prop, value) {
+            chrome.storage.sync.set({
+                [prop]: value
+            })
+        },
+        get: function(obj, prop) {
+            return SB.localconfig[prop]
+        }
+    };
+    return new Proxy({}, handler);
+}
+
+fetchConfig = _ => new Promise(function(resolve, reject) {
```

Comment:
Can you hange this to `(_) =>` or just `() =>` for clarity?

Reply:
Yes